.github/ISSUE_TEMPLATE.md

```diff
@@ -4,6 +4,8 @@ This issue tracker is only for technical issues related to bitcoin-core.
 
 General bitcoin questions and/or support requests and are best directed to the [Bitcoin StackExchange](https://bitcoin.stackexchange.com).
 
+For reporting security issues, please use [security@bitcoincore.org](mailto:security@bitcoincore.org).
```

Comment:
Indeed. The discussion belongs there, and so does the documentation. Maybe refer to the instructions for reporting security issues on the contact page: https://bitcoincore.org/en/contact/ instead of mentioning the address directly?
If we then happen to have GPG set up, it can be mentioned there without having to put everything into this template.